src/vm/codeman.cpp

```diff
@@ -3020,7 +3020,7 @@ void * EEJitManager::allocCodeFragmentBlock(size_t blockSize, unsigned alignment
     {
         CrstHolder ch(&m_CodeHeapCritSec);
 
-        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(TADDR), blockSize, alignment, &pCodeHeap);
+        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(CodeHeader), blockSize, alignment, &pCodeHeap);
```

Comment:
We only use the last field of CodeHeader on this codepath, so I think the original code was fine. We will allocate extra unused 12 bytes on Windows x86 with this change. It would be worth a comment at least.